azure-pipelines.yml

```diff
@@ -37,7 +39,7 @@ stages:
   jobs:
```

Comment:
What's the consideration that `FhirToDataLake` is built on ubuntu, while it is released on windows?

Reply:
Actually this PR mainly covers the build pipeline. Changes need be made in future to fix the release pipeline. Both pipelines should run in Linux platform.